sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -165,12 +165,16 @@
 MAX_DURATION_VALUE = 922337203685477
 # equivalent to .NET Date("9999-12-31T07:59:59.000Z").getTime() in ms
 MAX_ABSOLUTE_EXPIRY_TIME = 253402243199000
-
+MESSAGE_STATE_NAME = b"x-opt-message-state"
 
 class ServiceBusReceiveMode(str, Enum):
     PEEK_LOCK = "peeklock"
     RECEIVE_AND_DELETE = "receiveanddelete"
 
+class ServiceBusMessageState(str, Enum):
+    ACTIVE = "Active"
+    DEFERRED = "Deferred"
+    SCHEDULED = "Scheduled"
```

Comment:
I remember enum values should be of lower case?

```suggestion
    ACTIVE = "active"
    DEFERRED = "deferred"
    SCHEDULED = "scheduled"
```